napa.gemspec

```diff
@@ -9,9 +9,9 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://tech.bellycard.com"
   gem.licenses      = ['MIT']
 
-  gem.files         = `git ls-files`.split($\)
+  gem.files         = Dir['**/*'].keep_if { |file| File.file?(file) }
   gem.executables   << 'napa'
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+  gem.test_files    = Dir['{test,spec,features}/**/*']
```

Comment:
I'll remove test and features since napa doesn't have these.
Will squash commits today and it should be ready to merge